src/scripts/main.js

```diff
@@ -1,3 +1,197 @@
 'use strict';
 
-// write your code here
+const table = document.querySelector('table');
+let score = 0;
+
+let board = new Array(4)
+  .fill(null)
+  .map(el => [0, 0, 0, 0]);
+
+const rows = 4;
+const columns = 4;
+
+const button = document.querySelector('button');
+const messageLose = document.querySelector('.message-lose');
+const messageWin = document.querySelector('.message-win');
+const messageStart = document.querySelector('.message-start');
+
+button.addEventListener('click', () => {
+  if (button.classList.contains('start')) {
+    button.classList.remove('start');
+    button.classList.add('restart');
+    button.innerText = 'Restart';
+
+    messageStart.classList.add('hidden');
+  } else if (button.classList.contains('restart')) {
+    board = new Array(4).fill([0, 0, 0, 0]);
```

Comment:
You forgot to fix this value and after restarting, I can see 8 field cells instead of 2